test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -913,6 +924,16 @@ UCS_TEST_P(test_ucp_sockaddr, ep_query) {
     ep_query();
 }
 
+UCS_TEST_P(test_ucp_sockaddr, set_local_sockaddr)
+{
+    if (!m_test_addr.is_rdmacm_netdev()) {
```

Comment:
IMO no need to skip
maybe need to implement setting local address in TCP connection manager as well?

Reply:
changed accordingly